export-csv.js

```diff
@@ -42,7 +42,7 @@
                         xTitle = 'DateTime';
                     } else if (series.xAxis.categories) {
                         xData = Highcharts.map(xData, function (x) {
-                            return Highcharts.pick(series.xAxis.categories[x], x);
+                            return Highcharts.pick(series.data[x].name, x);
```

Comment:
This was changed in https://github.com/Appboy/platform/commit/6bec957e34832ce3efb3d2bb0f146e596389c909#diff-1c338b2d1560f83f0b76cd18eec26112